build/helpers/moment.js

```diff
@@ -0,0 +1,11 @@
+'use strict';
+
+var moment = require('moment');
+
+module.exports = function (context, options) {
+	var date = options ? context : new Date();
+	var options = options || context;
+	var format = options.hash.format || 'YYYY-MM-DD ';
```

Comment:
Local node with `moment`:

```
> moment().format('MMMM Do YYYY, h:mm:ss a');
'April 27th 2015, 1:52:18 pm'
> moment().utc().format('MMMM Do YYYY, h:mm:ss a');
'April 27th 2015, 8:52:42 pm'
```

This is what I'd expect to see but doesn't help to clarify @tylersticka 's claimed behavior.